.github/workflows/CI-Operator.yml

```diff
@@ -0,0 +1,150 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+name: C/I-Operator
+on: workflow_dispatch
+
+env:
+  EKS_ARM_64_AMP_ENDPOINT: "https://aps-workspaces.us-east-2.amazonaws.com/workspaces/ws-1de68e95-0680-42bb-8e55-67e7fd5d0861"
+  EKS_ARM_64_CLUSTER_NAME: "integ-test-arm64-cluster"
+  EKS_ARM_64_REGION: "us-east-2"
+  TESTING_FRAMEWORK_REPO: aws-observability/aws-otel-test-framework
+  # DDB_TABLE_NAME: BatchTestCache
```

Comment:
Is this not needed here?

Reply:
Meant to take this out, as this is used in the `validate-all-tests-pass` job which is used to set `release-candidate-ready`, but if `run-batch-jobs` is green that should indicate all tests passing as well